.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Build PR
+
+on: pull_request
```

Comment:
could run this on push too

Reply:
I think from now on changes should go through PRs, so I'm not sure what benefit we'd have to build on push.